tests/base.py

```diff
@@ -538,12 +534,11 @@ def request(self, path='/', method='GET', params=None, user=None,
             body = self.getBody(response)
             try:
                 response.json = json.loads(body)
-            except ValueError:
-                raise AssertionError('Did not receive JSON response')
+            except Exception:
```

Comment:
This should be able to stay as just `ValueError`, since `json.loads` raises a `ValueError` in Python 2 and a [`JSONDecodeError`](https://docs.python.org/3/library/json.html#json.JSONDecodeError) (which is a subclass of `ValueError`) in Python 3.

Is there some other type of exception that needs to be handled here?